packages/nuxt/package.json

```diff
@@ -84,10 +84,12 @@
       "build:transpile": {
         "dependsOn": [
           "^build:transpile",
-          "^build:types"
+          "^build:types",
+          "build:types"
```

Comment:
Transpilation usually shouldn't depend on the same packages types. Is there a reason you added this?

Reply:
This is because I am building the module with `nuxt-module-build` and this already relies on the types. I am getting the build warning `Potential missing package.json files: build/types/index.types.d.ts`.